app/__init__.py

```diff
@@ -99,7 +105,15 @@ def create_app(application, config=None):
     aws_sns_client.init_app(application, statsd_client=statsd_client)
     aws_ses_client.init_app(application.config["AWS_REGION"], statsd_client=statsd_client)
     notify_celery.init_app(application)
-    signer.init_app(application)
+
+    signer_notification.init_app(application, secret_key=application.config["SECRET_KEY"], salt="notification")
+    signer_personalisation.init_app(application, secret_key=application.config["SECRET_KEY"], salt="personalisation")
+    signer_complaint.init_app(application, secret_key=application.config["SECRET_KEY"], salt="complaint")
+    signer_delivery_status.init_app(application, secret_key=application.config["SECRET_KEY"], salt="delivery_status")
+    signer_bearer_token.init_app(application, secret_key=application.config["SECRET_KEY"], salt="bearer_token")
+    signer_api_key.init_app(application, secret_key=application.config["SECRET_KEY"], salt="api_key")
+    signer_inbound_sms.init_app(application, secret_key=application.config["SECRET_KEY"], salt="inbound_sms")
```

Comment:
Ah I forgot about these init_app calls... I don't like them because they leave an object in an invalid state after the ctor was called, waiting for proper initialization. But anyway, good for our purpose..

Reply:
yeah, right now the way we do it we need the application configured to get the `SECRET_KEY`. 🙁